CHANGELOG.md

```diff
@@ -5,6 +5,35 @@ file is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and
 this project adheres to [Semantic
 Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Changed
+
+- Configuration options required for establishing a connection with the central
+  management server must be specified directly via command-line interface.
+  The following options and arguments are required:
+  - `--cert`: Specifies the certificate file for the current module.
+  - `--key`: Specifies the private key file for the certificate.
+  - `--ca-certs`: Specifies the CA certificate files. Multiple CA certificates
+    can be provided by using this option multiple times.
+  - The manager server's name and address must be provided as a command-line
+    argument in the format `<server_name>@<server_ip>:<server_port>`.
+- The configuration file, previously provided as a positional command-line
+  argument, must now be specified using the `-c` option. This change allows for
+  additional command-line arguments to be used alongside the configuration file.
+  However, providing a local configuration file is optional.
+- If log_dir is not specified in the configuration file or the log file cannot
+  be created, logs will be printed to stdout.
+- Default last_timestamp_data file: /usr/local/aice/conf/time_data.json
+
+### Removed
+
+- Removed cert, key, ca_certs, review_name and review_rpc_srv_addr from the
+  configuration file; now mandatory via CLI.
+- Removed OS-specific configuration directory.
+  - Linux: _$HOME/.config/crusher/config.toml_
+  - macOS: _$HOME/Library/Application Support/com.cluml.crusher/config.toml_
```

Comment:
Could you explain why these are enclosed by `_`?

Reply:
The Notion document about using backtick said that backticks should be used for program code, command and argument.
I thought the file path is not one of them but wanted to emphasize it.

Could you clarify the usage for file path?

Reply:
File paths or anything else should be enclosed in backticks if they could appear in code or terminal commands. If they aren’t intended to be used in code or commands, I prefer leaving them in plain text. When it's unclear whether they might be used that way, either approach is fine. In my case, if uncertain, I prefer not using backticks.

Regardless of backticks, in CHANGELOG.md, `_` doesn't seem common to me.

Reply:
Removed the underscores, since the paths aren't used in code or commands.

I initially used underscores to italicize the paths but didn't realize this is uncommon in change logs.
Thank you.